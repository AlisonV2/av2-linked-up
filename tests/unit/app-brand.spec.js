import { shallowMount } from '@vue/test-utils';
import AppBrand from '@/components/nav/Brand.vue';

describe('AppBrand.vue', () => {
  it('AppBrand Snapshot', () => {
    const component = shallowMount(AppBrand);
    expect(component.element).toMatchSnapshot();
  });

  it('Checks brand image class', () => {
    const component = shallowMount(AppBrand);
    expect(component.text()).toContain('Linked Up');
  });
});
