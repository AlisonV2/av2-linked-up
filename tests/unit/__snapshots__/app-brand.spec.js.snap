// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBrand.vue AppBrand Snapshot 1`] = `
<div
  data-v-app=""
>
  
  <img
    class="navbar-brand desktop-logo"
    src="/img/logo-light.png"
  />
  <img
    class="navbar-brand mobile-logo"
    src="/img/logo.png"
  />
  <h1
    class="tablet-logo"
  >
    Linked Up
  </h1>
  
</div>
`;
